runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11969,9 +11969,10 @@ J9::Z::TreeEvaluator::VMinlineCompareAndSwap(TR::Node *node, TR::CodeGenerator *
    // compression sequence.
    bool isValueCompressedReference = false;
 
-   TR::Node* decompressedValueNode = newVNode;
+   static bool UseOldCompareAndSwapObject = (bool)feGetEnv("TR_UseOldCompareAndSwapObject");
```

Comment:
I understand the intention was to match the environment set throughout the code base, but we should rename variable name `useOldCompareAndSwapObject`

Reply:
I can change this.